src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -574,7 +580,8 @@ private static string GenerateRowField(string val, int width, int alignment, Dis
             return s;
         }
 
-        private const string ellipsis = "...";
+        private const char Ellipsis = '\u2026';
+        private const int EllipsisSize = 1;
```

Comment:
Should it be `EllipsisSize = sizeof(Ellipsis )`?

Reply:
sizeof(char) is 2 (Unicode)

Reply:
Size in the name confuses me. 😕